package/MDAnalysis/coordinates/DLPoly.py

```diff
@@ -154,9 +155,9 @@ def __init__(self, filename, **kwargs):
         super(HistoryReader, self).__init__(filename, **kwargs)
 
         # "private" file handle
-        self._file = open(self.filename, 'r')
-        self.title = self._file.readline().strip()
-        self._levcfg, self._imcon, self.n_atoms = np.int64(self._file.readline().split()[:3])
+        self._f = util.anyopen(self.filename, 'r')
```

Comment:
We'll probably want to document somewhere (in the AnalysisBase docstring at the very least?) that we expect files to be stored under self._f for the readers / writers.

Reply:
If it's important, make the attribute name more expressive, e.g., `_file` or `_stream` or `_open_stream`. Readability is important!